src/components/HeaderTabs/Tab/Tab.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { HeaderTab } from './style';
+
+const Tab = ({
+  title, value, index, onClickTab,
+}) => (
+  <HeaderTab
+    onClick={() => onClickTab(index)}
```

Comment:
You can pass `onClickTab` function as is.
```
onClick={onClickTab}
```

Reply:
At this place I just called function with ```index``` parameter. If I replace it with ```onClick={onClickTab}``` then it will not work. Or did I misunderstand?

Reply:
I made some corrections to the code. I was wondering if you could review it.